### PR TITLE
searchsploit: edit page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/searchsploit.md [hunk 1]
```diff
@@ -22,8 +22,16 @@
 
 - Open the resource to read with the pager defined in the `$PAGER` environment variable:
```
Comment (vitorhcl) [inline pages/common/searchsploit.md:23]: We can highlight the short option "-x":

```suggestion
- Open (e[x]amine) the resource, reading with the pager defined in the `$PAGER` environment variable:
```

Or even:

```suggestion
- E[x]amine the resource, using the pager defined in the `$PAGER` environment variable:
```

File: pages/common/searchsploit.md [hunk 2]
```diff
@@ -22,8 +22,16 @@
...
 
-`searchsploit --explore {{exploit_number}}`
+`searchsploit --examine {{exploit_number}}`
 
 - Update the local exploit database:
```
Comment (vitorhcl) [inline pages/common/searchsploit.md:27]:
```suggestion
- [u]pdate the local exploit database:
```
We can add this missing mnemonic

File: pages/common/searchsploit.md [hunk 3]
```diff
@@ -22,8 +22,16 @@
...
 
 `searchsploit --update`
+
+- Search for [c]ommon [v]ulnerabilities and [e]xposures (CVE) value:
```
Comment (vitorhcl) [inline pages/common/searchsploit.md:31]:
```suggestion
- Search for a [c]ommon [v]ulnerabilities and [e]xposures (CVE) value:
```

File: pages/common/searchsploit.md [hunk 4]
```diff
@@ -12,18 +12,26 @@
 
 `searchsploit sudo 1.8.27`
 
-- Show the exploit-db link to the found resources:
+- Show the exploit-db link (--[w]ww) to the found resources:
```
Comment (vitorhcl) [inline pages/common/searchsploit.md:15]: I didn't find a better way to mention the short option

File: pages/common/searchsploit.md [hunk 5]
```diff
@@ -12,18 +12,26 @@
...
 
 `searchsploit --www {{search_terms}}`
 
-- Make a copy of the resource to the current directory (requires the number of the exploit):
+- Copy ([m]irror) the resource to the current directory (requires the number of the exploit):
```
Comment (vitorhcl) [inline pages/common/searchsploit.md:19]: I think it's good to still leave copy as it's more understandable

File: pages/common/searchsploit.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # searchsploit
 
-> Searchsploit searches exploit database's database for exploits, shellcodes and/or papers.
+> Search Exploit Database's database for exploits, shellcodes and/or papers.
```
Comment (vitorhcl) [inline pages/common/searchsploit.md:3]: What do you guys think?

**Review by vitorhcl — COMMENTED** [review]
LGTM, thank you for your contribution! I have some suggestions

File: pages/common/searchsploit.md (hunk 6 above)
Comment (kbdharun) [reply pages/common/searchsploit.md:3]:
```suggestion
> Search Exploit's database for exploits, shellcodes and/or papers.
```
Maybe this?

File: pages/common/searchsploit.md (hunk 4 above)
Comment (kbdharun) [reply pages/common/searchsploit.md:15]:
```suggestion
- Show the exploit-db link to the found resources:
```
I don't think the short option is required here. If you really need it, then you can go with this:

```suggestion
- Show the exploit-db link to the found resources ([w]):
```

File: pages/common/searchsploit.md [hunk 7]
```diff
@@ -16,14 +16,22 @@
...
 
-`searchsploit --explore {{exploit_number}}`
+`searchsploit --examine {{exploit_number}}`
 
-- Update the local exploit database:
+- [u]pdate the local exploit database:
 
 `searchsploit --update`
+
+- Search for a [c]ommon [v]ulnerabilities and [e]xposures (CVE) value:
```
Comment (kbdharun) [inline pages/common/searchsploit.md:31]:
```suggestion
- Search for the [c]ommon [v]ulnerabilities and [e]xposures (CVE) value:
```

**Review by kbdharun — APPROVED** [review]

**Review by vitorhcl — APPROVED** [review]
LGTM, thank you for your contribution!

**Pull request merged**